examples/react-17/package.json

```diff
@@ -19,8 +19,8 @@
   },
   "devDependencies": {
     "@algolia/client-search": "4.16.0",
-    "@types/react": "^17.0.0",
-    "@types/react-dom": "^17.0.0",
+    "@types/react": "^19.0.7",
+    "@types/react-dom": "^19.0.3",
```

Comment:
this means the folder name should be changed to 19?

Reply:
Not if we don't update React to 19. Types are compatible between versions still.
Should be very quick to update though

Reply:
ah ok all good then

Reply:
Checked a bit more, they're not aptly named, `react` is an example with `autocomplete-js` while `react-17` is an example of a custom renderer with `autocomplete-core`.
Afraid if I change the name of the folder it'll break links on the docs, I think we can let it like this for now, esp considering what we'll do with the lib.